pkg/ansibleapp/types_test.go

```diff
@@ -16,17 +16,20 @@ const SpecDescription = "A note taking webapp"
 
 var SpecJSON = fmt.Sprintf(`
 {
-  "id": "%s",
+	"id": "%s",
 	"description": "%s",
-  "name": "%s",
-  "bindable": %t,
-  "async": "%s"
+	"name": "%s",
+	"bindable": %t,
+	"async": "%s"
```

Comment:
Ok, makes sense. Usually these types of changes are mistakes, but this one was on purpose. +1